e2etests/e2e_test.go

```diff
@@ -164,7 +164,9 @@ func run(t *testing.T, tc testCase) {
 		pathGotSVG := filepath.Join(dataPath, "sketch.got.svg")
 
 		svgBytes, err := d2svg.Render(diagram, &d2svg.RenderOpts{
-			Pad: d2svg.DEFAULT_PADDING,
+			Pad:         d2svg.DEFAULT_PADDING,
+			ThemeID:     0,
+			DarkThemeID: math.MaxInt64,
```

Comment:
why maxint?

Reply:
MaxInt64 means no dark theme preference, i.e. no `@media` query is created. I couldn't use `0` because it is an already existing theme. When `DarkThemeID` is set to `0`, the `@media` query would be added for the theme with id `0`.

Reply:
is that alright? did I explain it correctly?

Reply:
i understand. i left a relevant comment elsewhere, perhaps missed because of github UI collapsing: https://github.com/terrastruct/d2/pull/613#discussion_r1090052701